plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -118,6 +118,7 @@ public TransactionLogAccess(
                 .build();
         activeDataFileCache = EvictableCacheBuilder.newBuilder()
                 .weigher((Weigher<String, DeltaLakeDataFileCacheEntry>) (key, value) -> Ints.saturatedCast(estimatedSizeOf(key) + value.getRetainedSizeInBytes()))
+                .expireAfterWrite(deltaLakeConfig.getDataFileCacheTtl().toMillis(), TimeUnit.MILLISECONDS)
```

Comment:
I can think of a test where we'd create a table, add some content to it, wait for a few millis more than the expiration time of the active data files cache and then check through `AccessTrackingFileSystem` whether the data files of the tables  are opened again  - however i'm not sure of the added value of the test. Feedback welcome.